crates/starknet-devnet/tests/test_simulate_transactions.rs

```diff
@@ -232,6 +234,204 @@ mod simulation_tests {
         );
     }
 
+    async fn update_gas_scenario(devnet: BackgroundDevnet, assert_chain_id: &str) {
```

Comment:
IMO `assert_chain_id` is a misleading name; it should be `expected_chain_id`.

Reply:
renamed in 6a757fc2d3224524adeda637d68da8423cc81476